src/portalbackend/PortalBackend.DBAccess/Repositories/OfferSubscriptionsRepository.cs

```diff
@@ -500,4 +500,24 @@ public void AttachAndModifyAppSubscriptionDetail(Guid detailId, Guid subscriptio
                 os.ConnectorAssignedOfferSubscriptions.Select(c => c.ConnectorId)
             ))
             .ToAsyncEnumerable();
+
+    /// <inheritdoc>
+    public IAsyncEnumerable<OfferSubscriptionStatusData> GetOwnCompanySubscribedAppsOfferSubscriptionStatusesUntrackedAsync(Guid userCompanyId, OfferTypeId offerTypeId, DocumentTypeId documentTypeId) =>
```

Comment:
Since we can pass the offerType we should change the naming

```suggestion
    public IAsyncEnumerable<OfferSubscriptionStatusData> GetOwnCompanySubscribedOffersOfferSubscriptionStatusesUntrackedAsync(Guid userCompanyId, OfferTypeId offerTypeId, DocumentTypeId documentTypeId) =>
```